hikari/presences.py

```diff
@@ -118,11 +121,16 @@ class ActivityParty:
     """Maximum size of this party, if applicable."""
 
 
+_DYNAMIC_URLS = {"mp": urls.MEDIA_PROXY_URL + "/{}"}
+
+
 @attr_extensions.with_copy
 @attr.define(hash=False, kw_only=True, weakref_slot=False)
 class ActivityAssets:
     """Used to represent possible assets for an activity."""
 
+    _application_id: typing.Optional[snowflakes.Snowflake] = attr.field(repr=False)
```

Comment:
Should this be public?

Reply:
why would you it need to be public

Reply:
If its something they send, why not give it to the user too?

Reply:
It's from the parent object

Reply:
Oh sorry, completely missed that